docs/standard/design-guidelines/choosing-between-anonymous-and-tuple.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Choosing between anonymous and tuple types"
+description: Learn how to decide whether to design a type as a class, or to design a type as a struct.
+ms.date: 06/24/2020
+ms.technology: dotnet-standard
+---
+
+# Choosing between anonymous and tuple types
+
+As a developer, choosing the appropriate type can be scary task. Anonymous types have been available since C# 3.0, while generic <xref:System.Tuple%602?displayProperty=nameWithType> types were introduced with .NET Framework 4.0. There are more modern options that exist, such as <xref:System.ValueTuple%602?displayProperty=nameWithType> - which as the name implies, provide a value type with the flexibility of anonymous types. In this article, you'll learn when it's appropriate to choose one type over the other.
```

Comment:
I'd rework this paragraph.  I wouldn't use the word "scary", but it does involve tradeoffs.

Secondly, I wouldn't refer to Tuples as "more modern". They were added in a later release in C#, but the concepts have been around for longer.